superset/sqllab/api.py

```diff
@@ -70,17 +73,70 @@ class SqlLabRestApi(BaseSupersetApi):
 
     class_permission_name = "SQLLab"
 
+    estimate_model_schema = EstimateQueryCostSchema()
     execute_model_schema = ExecutePayloadSchema()
 
     apispec_parameter_schemas = {
         "sql_lab_get_results_schema": sql_lab_get_results_schema,
     }
     openapi_spec_tag = "SQL Lab"
     openapi_spec_component_schemas = (
+        EstimateQueryCostSchema,
         ExecutePayloadSchema,
         QueryExecutionResponseSchema,
     )
 
+    @expose("/estimate/", methods=["POST"])
+    @protect()
+    @statsd_metrics
+    @requires_json
+    @permission_name("get")
```

Comment:
sorry, having second thoughts on this, let's remove it and leave the permission has it is. `can estimate query cost on SQLLab`. Also add that permission here: https://github.com/apache/superset/blob/master/superset/security/manager.py#L253

Reply:
is that a valid permission? I only see `allows_cost_estimate` as a boolean on the Database model?

Reply:
it should be can you find `estimate_query_cost` on ab_permission table?